### ISSUE TITLE
Metamask not able to connect to Main Ethereum Network, not able to login. Loading, infinite spinner

### ISSUE THREAD

**Issue by amirsohail01** [MetaMask/metamask-extension#3151]
![image](https://user-images.githubusercontent.com/12250521/35598850-d6ae86f8-0679-11e8-925a-e1bc0bf5cf48.png)

Not able to get past this error. I changed the network in the top left menu to another network and back again - didn't work. Uninstalled/Re-installed the extension in Chrome - didn't work, installed the extension in Mozilla and same thing, Works fine on my other PC.

**Comment by vimatrel** [comment]
Same issue here. Just installed the latest version on a new macbook and can't connect

**Comment by manecas8** [comment]
Same issue here. :-(

**Comment by danfinlay** [comment]
This is a known issue, duplicate of #2548.

You can work around it until we fix by switching the network (top left menu) to another network and back again.

**Comment by danfinlay** [comment]
Could you look in the background inspector, under the network tab, to see what errors you’re getting?

https://github.com/MetaMask/faq/blob/master/LOGS.md#background-logs-chrome

**Comment by amirsohail01** [comment]
![image](https://user-images.githubusercontent.com/12250521/35832291-d7cbe1d8-0b20-11e8-8610-5d970942d65b.png)

Hi Dan, please find the attached image.

**Comment by danfinlay** [comment]
It looks like you’re likely to be behind a firewall that is preventing outgoing connections to Infura. Maybe @maurcy would have more ideas.

**Comment by amirsohail01** [comment]
thanks for the update Dan, will wait for @maurcy's diagnosis.

**Comment by amirsohail01** [comment]
still waiting :(

**Comment by danfinlay** [comment]
Sorry for the wait, did you take any time to look at whether you were hitting a firewall?

Another step you could take:  Switch to the `Network` tab in the background inspector, and look closer at some of these failed responses. I suspect you'll find they're errors from your router/firewall. This problem probably needs to be solved on your side.

**Comment by amirsohail01** [comment]
![image](https://user-images.githubusercontent.com/12250521/36182137-3bb5a362-117c-11e8-9c38-11b4da886d0e.png)

Here is a snapshot from Network tab. My concern is that it was always working on this PC on the same network for last so many months, and the problem surfaced only recently. If you can guide where I need to dig further, I'll do that accordingly.

**Comment by amirsohail01** [comment]
Clicking on each of the red ones gives out below detail, I guess this has to be the problem with my network i.e. the 'Status Code' is same for all 3 REDs, "403 Forbidden". 

If that's the right conclusion, please accept my gratitude for your help and feel free to close this ticket. 

Cheers

![image](https://user-images.githubusercontent.com/12250521/36182286-069428e2-117d-11e8-801f-71740772d38f.png)

**Comment by danfinlay** [comment]
I asked Infura about this, and they don't see many 403s at all, so this really looks like a local networking issue. You will have to dig into your local network. Look at your router, the router it's connected to, the modem, etc.

You might get more detail if you clicked the `Response` tab within that single failed request's view.

**Comment by amirsohail01** [comment]
Spot on! It's my work PC and it appears that MM has been blocked on my work PC/network. Please feel free to close the ticket. 
Thanks for your help again :)

**Comment by redskate** [comment]
Same issue ...
